.github/workflows/release-reproducible-smart-contract.yaml

```diff
@@ -10,7 +10,6 @@ on:
 env:
   REGISTRY: docker.io
   SERVICE_NAME: 'verifiable-sc'
-  SOURCE_IMAGE_TAG: 'rust:1.75.0'
```

Comment:
The env variable `SOURCE_IMAGE_TAG` is still used in this workflow so that needs some clean-up
  
  ```     
            if [ $SERVICE_TAG != ${{ env.SOURCE_IMAGE_TAG }} ]; then
                echo "Tag version does not match the source image version."
                exit 1
            fi
   ```